src/collections/_documentation/workflow/issue-owners.md

```diff
@@ -3,57 +3,79 @@ title: 'Issue Owners'
 sidebar_order: 2
 ---
 
-The Issue Owners feature allows you to reduce noise by directing notifications to specific teams or users based on a path or URL. This allows you to get issues into the hands of the developer who can fix them, faster.
+The Issue Owners feature allows you to create rules to decide which user or team should own an [Issue]({%- link _documentation/data-management/event-grouping/index.md -%}). These rules resemble a typical code owners file in a repository, and can match on file paths of files in the stack trace, URL of the request, or event tags. You can automatically assign issues to their respective owners, which will trigger an alert, allowing you to find the developer with the most context about a fix.
```

Comment:
so... automatically assigning an issue to its owner doesn't trigger an alert, they're separate systems. maybe instead of "which will trigger an alert", say "or alert the owners about the issue"?